modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/ckeditor5/editor.scss

```diff
@@ -49,6 +49,10 @@
 		}
 	}
 
+	&.ck-content {
```

Comment:
Two comments on this:

- When opening the Styles dropdown with the new min-height the layout breaks
<img width="937" height="505" alt="image" src="https://github.com/user-attachments/assets/266f079b-f5b0-493e-ae58-f66dab039d46" />

- When changing to Source mode, the min-height no longer applies:
<img width="973" height="350" alt="image" src="https://github.com/user-attachments/assets/417e76d1-e8c5-4b9c-a9b0-1bbc26e1fc52" />

Reply:
alrighty, lemme look into it and fix